koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -1001,18 +1001,27 @@ def populate_openshift_cluster_information_tables(self, provider, cluster_id, cl
     def populate_cluster_table(self, provider, cluster_id, cluster_alias):
         """Get or create an entry in the OCP cluster table."""
         with schema_context(self.schema):
-            cluster, created = OCPCluster.objects.get_or_create(
-                cluster_id=cluster_id, cluster_alias=cluster_alias, provider_id=provider.uuid
-            )
+            cluster = OCPCluster.objects.filter(provider_id=provider.uuid).first()
+            msg = f"Fetching entry in reporting_ocp_clusters for provider: {provider.uuid}"
```

Comment:
This log message should come first:
```suggestion
            LOG.info(log_json(msg="fetching entry in reporting_ocp_cluster", provider_uuid=provider.uuid))
            cluster = OCPCluster.objects.filter(provider_id=provider.uuid).first()
            msg = "fetched entry in reporting_ocp_cluster"
```